documentation/docs/clients/saml.md

```diff
@@ -274,3 +274,81 @@ SimpleSAMLphp exposes his IdP metadata on `http://idp-domain/simplesamlphp/saml2
 	</md:EntityDescriptor>
 </md:EntitiesDescriptor>
 ```
+
+### Custom OpenSAML bootstrap
```

Comment:
Could you also add a line about this new feature in the "Release notes" doc: https://github.com/pac4j/pac4j/blob/master/documentation/docs/release-notes.md?

Reply:
I've added a note in the release notes